src/wallet/rpcwallet.cpp

```diff
@@ -2358,8 +2358,8 @@ static UniValue setcoinjoinamount(const JSONRPCRequest& request)
         "\nSet the goal amount in " + CURRENCY_UNIT + " for CoinJoin.\n",
         {
             {"amount", RPCArg::Type::NUM, RPCArg::Optional::NO,
-                "The default amount is " + std::to_string(DEFAULT_COINJOIN_AMOUNT) +
-                " Cannot be more than " + std::to_string(MAX_COINJOIN_AMOUNT) + " nor less than " + std::to_string(MIN_COINJOIN_AMOUNT)},
+                "The default amount is " + ToString(DEFAULT_COINJOIN_AMOUNT) +
+                " Cannot be more than " + ToString(MAX_COINJOIN_AMOUNT) + " nor less than " + ToString(MIN_COINJOIN_AMOUNT)},
```

Comment:
>can you tell me which commits they are so I can sort them out?

18449: these should be in 18134 ideally but, again, not a big deal imo

Reply:
Resolve with f4f6c6546337e318637f05fcc4aba5ed9a64195f